### PR TITLE
Mc toc param filter

### PR DESCRIPTION
**Pull request by FBRTMaka**

### CODE REVIEW

**Comment by DanielJMaher** [comment]
Let's hold until I  can get the filter terms!  What file would I find that in?

File: app.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def science(args):
 
     if app.config['DEBUG'] == True:
         app.add_url_rule('/site-map', 'site_map', site_map)
-    app.run(host='localhost', debug=True)
+    app.run(host='0.0.0.0', debug=True)
```
Comment (DanielJMaher) [inline app.py]: Just for my knowledge, why did you move this from localhost to broadcast?

Comment (FBRTMaka) [inline app.py]: good catch! that's not supposed to be in there.

**Comment by DanielJMaher** [comment]
OBE
